packages/grid_client/scripts/utils.ts

```diff
@@ -3,4 +3,17 @@ import { inspect } from "util";
 function log(message) {
   console.log(inspect(message, { showHidden: false, depth: null, colors: true }));
 }
-export { log };
+
+async function pingNodes(client, nodes) {
```

Comment:
Please define the types of `client` and `nodes`, also, would be better if we have a `DocString` for any new function/class created to provide the user/developer some details about what is the actual job that the function does, what params it takes and what it returns.